### PR TITLE
More musicbrainz fixes...

### PR DESCRIPTION
**Pull request by TimHenkelmann**
- make musicbrainz server.js accessible from the cli
- separated queryCounter from requestCounter in the callback
- adjusted musicbrainz scripts in cli
- proper variable renaming
- removed testoutput from getArtists
- fixed date issue when populating db
- added pseudonym and tags

### CODE REVIEW

File: musicbrainz/getArtists.js [hunk 1]
```diff
@@ -71,23 +83,46 @@ module.exports = {
...
                             name: name,
+                            artist_type: artist_type,
                             nationality: nationality,
                             dateOfBirth: dateOfBirth,
                             dateOfDeath: dateOfDeath,
                             placeOfBirth: placeOfBirth,
                             placeOfDeath: placeOfDeath,
                             instrument: null,
-                            psuedonym: null,
+                            psuedonym: pseudonym,
```
Comment (ShilpaGhanashyamGore) [inline musicbrainz/getArtists.js]: pseudonym is the spelling in the schema. Please change it so .Else DB population will fail

File: musicbrainz/getArtists.js [hunk 2]
```diff
@@ -71,23 +83,46 @@ module.exports = {
...
                             work: null,
                             release: null,
-                            tag: null,
+                            tag: tags,
```
Comment (ShilpaGhanashyamGore) [inline musicbrainz/getArtists.js]: Also here it is tags

**Review by ShilpaGhanashyamGore — APPROVED** [review]
I left some comments .With those fixes we can merge it

File: musicbrainz/getArtists.js (hunk 1 above)
Comment (metavaults) [reply musicbrainz/getArtists.js]: shouldn't it be null? We have no pseudonyms at this point, right?

**Review by metavaults — COMMENTED** [review]
Considering Shilpa's comments, great job! We can merge it.

File: musicbrainz/getArtists.js (hunk 1 above)
Comment (TimHenkelmann) [reply musicbrainz/getArtists.js]: @ShilpaGhanashyamGore Yep you're right, already changed that but forgot to commit it!
@LukasNavickas We do! If you check the scraped json, there is an element called "aliases"

**Pull request merged**